automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -351,5 +353,100 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike with
         }
       }
     }
+
+    "should have correct policies in Sam and ACLs in Google when workspace is created" in {
+      import DefaultJsonProtocol._
+      implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = 20 seconds)
+      implicit val token: AuthToken = ownerAuthToken
+      val workspacePolicyNames = Set("can-compute", "project-owner", "writer", "reader", "share-writer", "can-catalog", "owner", "share-reader")
+      // bucket and object access levels for sam policies as described in comments in insertBucket function in HttpGoogleServicesDAO
+      val policyToBucketAccessLevel = Map("project-owner" -> "WRITER", "owner" -> "WRITER", "writer" -> "WRITER", "reader" -> "READER")
+      val policyToObjectAccessLevel = Map("project-owner" -> "READER", "owner" -> "READER", "writer" -> "READER", "reader" -> "READER")
+      val serviceAccountEmailDomain = "developer.gserviceaccount.com"
+
+      withCleanBillingProject(owner) { projectName =>
+        withWorkspace(projectName, s"unconstrained-workspace") { workspaceName =>
+          val workspaces = Rawls.workspaces.list()
+          val workspaceId = workspaces.parseJson.asInstanceOf[JsArray].elements.flatMap { workspaceListEntry =>
+            workspaceListEntry.asJsObject.getFields("workspace").flatMap { workspace =>
+              workspace.asJsObject.getFields("workspaceId")
+            }
+          }.head.convertTo[String]
```

Comment:
there is no guarantee that the first workspace is the one you are looking for... find the workspace in the list with the right namespace (should match projectName) and name (should match workspaceName).